### DISCUSSION TITLE
Please add "byTestId" selector to framework

### DISCUSSION THREAD

**Discussion by mbuchner** [selenide/selenide#2734] [General]
First of all- thanks for the great framework - we use it a lot!

In the recent years an attribute has been "standardized" to be used to uniquely select elements "data-testid".

Could you add this selector to your framework?

```
$(By.testId("some-test-id")).setValue(query).pressEnter();

$(byTestId("some-test-id")).shouldBe(visible));
```

Like here: https://playwright.dev/java/docs/locators#locate-by-test-id

At the moment we use a helper function for it but I guess it would be easy for you to include it into selenide:
```
  public static String byTestId(String testid) {
    return "[data-testid='" + testid + "']";
  }
```

Thanks Max

**Comment by asolntsev** [comment]
@mbuchner Yes, it might be a good feature. 
The problem I see is that this is not so much "standardized". People might use `data-testid`, `data-test-id`, `data-testId`, `data-TestId`, `data-test` etc.

I see PW suggest a method to customize this attribute name: `playwright.selectors().setTestIdAttribute("data-pw");`

But is it really so much useful? 
Writing a test helper is not longer. In fact, you can write even shorter code in Selenide: `$(by("data-test-uuid", "12345"))`
